bin/verilator

```diff
@@ -3243,6 +3263,19 @@ Specifies that following lines of code should have coverage re-enabled (if
 appropriate --coverage flags are passed) after being disabled earlier with
 /*verilator coverage_off*/.
 
+=item /*verilator hier_block*/
+
+Specifies that the module is a unit of hierarchical verilation to reduce the
+runtime and the memory consumption for verilation. The module will not
+be inlined nor uniquified to each instance. The hierachical block has some
+limitations such as:
+
+   The block can not be accessed using dot (.) from upper module.
+   Signals in the block can not be traced.
+
+Modules marked by this metacomment may instantiate other modules that are
+also marked by this metacomment.
+
```

Comment:
Suggest we make a new head1 maybe right before MULTITHREADING to describe this, as think this will be a large topic.